### ISSUE TITLE
[Equalizer] reflow , init is buggy

### ISSUE THREAD

**Issue by designerno1** [zurb/foundation-sites#7423]
in the reflow (init) of Equalizer is
```js
$('[' + this.attr + ']').each(function() {
      var $eqParent       = $(this),
          adjustedHeights = [],
          $images = $eqParent.find('img');

      if ($images.length) {
        Foundation.onImagesLoaded($images, function() {
          adjustedHeights = self.getHeights($eqParent);
          self.applyHeight($eqParent, adjustedHeights);
        });
      }
      else {
        adjustedHeights = self.getHeights($eqParent);
        self.applyHeight($eqParent, adjustedHeights);
      }
    });
```
this causes the functions to execute as many data-equalizer elements you have in the markup with the wrong settings from the actual equalizer
Because there is the each function
for Example
you got 3 data-equalizer tags in the html so the plugin executes 3 tims and 3 time the each loop gets the heights and adjuststhe heights but with the settings from the wrong element

1. settings equalizeOnStack=true
2. settings equalizeOnStack=false
3. settings equalizeOnStack=true

so the 2. equalizer gets executed last in the each loop with settings equalizeOnStack=true what are not the settings of the 2. equalizer

so why the each loop is done in the reflow???

i hope its understandable ;-)

**Comment by designerno1** [comment]
i got the equalizeOnStack function ready to go and as a pull request, 
but with the buggy each reflow init, it only works if all equalizers in makup has the same settings, or there is only 1 equalizer in markup

**Comment by zurbchris** [comment]
Should be fixed with 6.1 equalizer. That equalize by row was a huge addition, btw. You Rock!
